app/src/main/java/net/graphmasters/navigation/example/MainActivity.kt

```diff
@@ -143,6 +144,8 @@ class MainActivity : AppCompatActivity(), LocationListener,
             this, Manifest.permission.ACCESS_FINE_LOCATION
         ) == PackageManager.PERMISSION_GRANTED
 
+    private var voiceInstructionComponent: VoiceInstructionComponent? = null
```

Comment:
To be consistent with the rest of the code move this up to the other components and make it `lateinit`